tests/lambda_functions/delivery_status_processor_lambda/test_delivery_status_processor_lambda.py

```diff
@@ -18,30 +18,34 @@ def all_path_env_param_set(monkeypatch):
 def event():
     """Generates a sample ALB received Twilio delivery status event object."""
     return {
-        "requestContext": {"elb": {"targetGroupArn": "<DEV TARGET GROUP>"}},
+        "requestContext": {
+            "elb": {
+                "targetGroupArn": ""
+            }
+        },
         "httpMethod": "POST",
-        "path": "/deliverystatus",
+        "path": "/sms/deliverystatus",
         "queryStringParameters": {},
         "headers": {
             "accept": "*/*",
             "connection": "close",
-            "content-length": "227",
+            "content-length": "277",
             "content-type": "application/x-www-form-urlencoded; charset=utf-8",
-            "host": "dev-api.va.gov",
-            "i-twilio-idempotency-token": "50609cf4-07f3-4e3f-ac42-044bd13bbc6c",
+            "host": "api.va.gov",
+            "i-twilio-idempotency-token": "32e709c9-0e95-47c0-8f0d-c6c868cf6241",
             "user-agent": "TwilioProxy/1.1",
-            "x-amzn-trace-id": "Self=<SOME VALUE>",
-            "x-forwarded-for": "<COMMA SEPARTED IPS>",
-            "x-forwarded-host": "dev-api.va.gov:443",
+            "x-amzn-trace-id": "",
+            "x-forwarded-for": "",
+            "x-forwarded-host": "api.va.gov:443",
             "x-forwarded-port": "443",
             "x-forwarded-proto": "https",
             "x-forwarded-scheme": "https",
             "x-home-region": "us1",
-            "x-real-ip": "<SOME IP>",
-            "x-twilio-signature": "<SOME VALUE>",
+            "x-real-ip": "",
+            "x-twilio-signature": "GV6ZwxO2f7qTuUKx94saVKju4XI=",
+            "x-use-static-proxy": "true"
         },
-        "body": "U21zU2lkPXRoaXNpc3NvbWVzbXNpZCZTbXNTdGF0dXM9c2VudCZNZXNzYWdlU3RhdHVzPXNlbnQmVG89JTJCMTExMTExMTExMTEmTWVzc2FnZVNpZD1zb21lbWVzc2FnZWlkZW50aWZpZXImQWNjb3VudFNpZD10d2lsaW9hY2NvdW50c2lkJkZyb209JTJCMjIyMjIyMjIyMiZBcGlWZXJzaW9uPTIwMTAtMDQtMDE=",
-        "isBase64Encoded": True,
+        "body": "TWVzc2FnaW5nU2VydmljZVNpZD1NR2JlNmJiMjdhOWNhMTVkZTI3NmM1Yjg2ZjBkOWU5YzJlJkFwaVZlcnNpb249MjAxMC0wNC0wMSZNZXNzYWdlU3RhdHVzPXF1ZXVlZCZTbXNTaWQ9U002ZGYzNDRlZDc3YmI2ZmEyMGI0ZDNhOWNhYjc3NWFjMyZTbXNTdGF0dXM9cXVldWVkJlRvPSUyQjE4NDU3MDVYWFhYJkZyb209NTMwNzkmTWVzc2FnZVNpZD1TTTZkZjM0NGVkNzdiYjZmYTIwYjRkM2E5Y2FiNzc1YWMzJkFjY291bnRTaWQ9QUM1NTFmYzYwODZmOTNhODM0OTI0NjZmYzYwNGM2OGFmNA=="
```

Comment:
the original body when decoded contains no sensitive info. the revised body does contain sensitive info. let's ask Kyle which he wants to use in tests.